src/metabase/integrations/ldap.clj

```diff
@@ -52,6 +52,10 @@
   (tru "Attribute to use for the user's email. (usually ''mail'', ''email'' or ''userPrincipalName'')")
   :default "mail")
 
+(defsetting ldap-attribute-uid
+  (tru "Attribute to use for the user's id. (usually ''uid'')")
```

Comment:
`user's` needs to be escaped here, like `uid` is escaped

Reply:
I just copied from the ldap-attribute-email, which is not escaped.